<ui version="4.0" >
 <class>TransportSetup</class>
 <widget class="QDialog" name="TransportSetup" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Public IM Accounts</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gb_aim" >
     <property name="title" >
      <string>AIM</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="lb_img_aim" >
        <property name="text" >
         <string>aimimg</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="2" >
         <widget class="QLabel" name="lb_euser_aim" >
          <property name="text" >
           <string>Example: johndoe12</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="lb_user_aim" >
          <property name="text" >
           <string>Screen Name:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="lb_pass_aim" >
          <property name="text" >
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="le_pass_aim" >
          <property name="echoMode" >
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="le_user_aim" />
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>51</width>
          <height>62</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_msn" >
     <property name="title" >
      <string>MSN</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="lb_img_msn" >
        <property name="text" >
         <string>msnimg</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="2" column="1" >
         <widget class="QLineEdit" name="le_pass_msn" >
          <property name="echoMode" >
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="le_user_msn" />
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="lb_user_msn" >
          <property name="text" >
           <string>Email Address:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="le_nick_msn" />
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="lb_pass_msn" >
          <property name="text" >
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="lb_euser_msn" >
          <property name="text" >
           <string>Example: johndoe@msn.com</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="lb_nick_msn" >
          <property name="text" >
           <string>Nickname:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>61</width>
          <height>95</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_yahoo" >
     <property name="title" >
      <string>Yahoo</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="lb_img_yahoo" >
        <property name="text" >
         <string>yahooimg</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <widget class="QLabel" name="lb_user_yahoo" >
          <property name="text" >
           <string>Yahoo ID:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="le_user_yahoo" />
        </item>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="le_pass_yahoo" >
          <property name="echoMode" >
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="lb_pass_yahoo" >
          <property name="text" >
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="lb_euser_yahoo" >
          <property name="text" >
           <string>Example: johndoe12</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>51</width>
          <height>62</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_icq" >
     <property name="title" >
      <string>ICQ</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="lb_img_icq" >
        <property name="text" >
         <string>icqimg</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="le_user_icq" />
        </item>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="le_pass_icq" >
          <property name="echoMode" >
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="lb_pass_icq" >
          <property name="text" >
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="lb_user_icq" >
          <property name="text" >
           <string>UIN:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>61</width>
          <height>62</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>504</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>gb_aim</tabstop>
  <tabstop>le_user_aim</tabstop>
  <tabstop>le_pass_aim</tabstop>
  <tabstop>gb_msn</tabstop>
  <tabstop>le_user_msn</tabstop>
  <tabstop>le_nick_msn</tabstop>
  <tabstop>le_pass_msn</tabstop>
  <tabstop>gb_yahoo</tabstop>
  <tabstop>le_user_yahoo</tabstop>
  <tabstop>le_pass_yahoo</tabstop>
  <tabstop>gb_icq</tabstop>
  <tabstop>le_user_icq</tabstop>
  <tabstop>le_pass_icq</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
